docs/Getting Started/React Native/index.md

```diff
@@ -0,0 +1,16 @@
+# React Native
```

Comment:
Change to "Can I Integrate SDL into a React Native App?" and redesign to answer that question.